pandas/core/indexes/base.py

```diff
@@ -2611,7 +2620,7 @@ def drop_duplicates(self, keep="first"):
 
         return super().drop_duplicates(keep=keep)
 
-    def duplicated(self, keep="first"):
+    def duplicated(self, keep="first") -> np.ndarray:
```

Comment:
keep is str?

Reply:
It can accept a str or a literal `False`. I've set the type to `Union[str, bool]`.

Reply:
For a different PR, bit I don't like the `bool` type, that can not accept `True`. I'd like to deprecate the bool type and accept only string parameter options, e.g. `"none"` instead of False. Any comment on that?